README.md

```diff
@@ -45,7 +45,7 @@ The guidelines described below are based on:
   │   ├── home-dashboard.component.spec.ts
   │   ├── login.component.ts
```

Comment:
016-02-09T04:29:24+07:00,Membership,Invited a team member,203.205.15.216,VN,Oh Kendy,kendyoh1989@gmail.com,"{""email"": ""leeoh369@gmail.com""}"
2016-02-09T04:21:18+07:00,Logins,Signed in,203.205.15.216,VN,Oh Kendy,kendyoh1989@gmail.com,
2016-02-09T04:14:44+07:00,Sharing,Opened a link (non-team member),203.205.15.216,VN,N/A,N/A,"{""base_name"": ""31343430383939373937303231"", ""link_owner_name"": ""Oh Kendy"", ""link_owner_member_id"": ""dbmid:AACzPggwpjY2h9BYvpMz16WQ2cTXbyNEBOw"", ""file_id"": null, ""link_owner_id"": 456290418, ""link_owner_email"": ""kendyoh1989@gmail.com"", ""path"": ""/Auto Call Recorder/.metadata/31343430383939373937303231""}"
2016-02-09T04:14:43+07:00,Sharing,Opened a link (non-team member),203.205.15.216,VN,N/A,N/A,"{""base_name"": ""31343430383939373937303231"", ""link_owner_name"": ""Oh Kendy"", ""link_owner_member_id"": ""dbmid:AACzPggwpjY2h9BYvpMz16WQ2cTXbyNEBOw"", ""file_id"": null, ""link_owner_id"": 456290418, ""link_owner_email"": ""kendyoh1989@gmail.com"", ""path"": ""/Auto Call Recorder/.metadata/31343430383939373937303231""}"
2016-02-09T04:12:45+07:00,Sharing,Downloaded the contents of a link (non-team member),203.205.15.216,VN,N/A,N/A,"{""base_name"": ""31343430383939373937303231"", ""link_owner_name"": ""Oh Kendy"", ""link_owner_member_id"": ""dbmid:AACzPggwpjY2h9BYvpMz16WQ2cTXbyNEBOw"", ""file_id"": null, ""link_owner_id"": 456290418, ""link_owner_email"": ""kendyoh1989@gmail.com"", ""path"": ""/Auto Call Recorder/.metadata/31343430383939373937303231""}"
2016-02-09T04:12:34+07:00,Sharing,Downloaded the contents of a link (non-team member),203.205.1